libs/messenger-bundle/composer.json

```diff
@@ -0,0 +1,72 @@
+{
+    "name": "keboola/messenger-bundle",
+    "description": "Keboola Messenger Bundle",
+    "license": "MIT",
+    "type": "symfony-bundle",
+    "authors": [
+        {
+            "name": "Keboola",
+            "email": "devel@keboola.com"
+        }
+    ],
+
+    "autoload": {
+        "psr-4": {
+            "Keboola\\MessengerBundle\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Keboola\\MessengerBundle\\Tests\\": "tests/"
+        }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:keboola/MessengerAzureBundle.git"
+        }
+    ],
+
+    "require": {
+        "php": ">=8.2",
+        "aymdev/messenger-azure-bundle": "dev-dsnParser",
```

Comment:
Schvalne tu je dev branch. Musel jsem v te knihovne torchu poupravit inicializaci, aby to fungovalo stejne jako treba AWS. Uz driv jsme nejake upravy do te knihovny posilal, autor je v pohode prijal, tak doufam, ze i tohle by mohlo jit hladce, jen to treba chvili potrva https://github.com/AymDev/MessengerAzureBundle/pull/14

Zmeny jsem poslal v PR do upstreamu (https://github.com/AymDev/MessengerAzureBundle/pull/14),

Reply:
V cem to bude lepsi? Lock na commit ti pak resi `composer.lock` v appce. Zadne libky nezamykame explicitne na konkretni commit a do bug-fix tagu (ktery automaticky instalujeme pri update) muze taky kdokoliv napchat cokoliv.

Reply:
Btw. ono to vlastne vubec neinstaluje "z ciziho" repa, ale z naseho forku (viz. `repositories` vys).